docs/Explanation/ADR/adr-009-fungibility.md

```diff
@@ -0,0 +1,96 @@
+# ADR 009: Fungibility
+
+## Changelog
+
+- 2021-12-03: Initial draft
+
+## Status
+
+PROPOSED
+
+> For details on ADR workflow, see the [PROCESS](./PROCESS.md#adr-status) page.
+> Use DRAFT if the ADR is in a draft stage (draft PR) or PROPOSED if it's in review.
+
+## Abstract
+
+It is anticipated that Elesto will have more than one Issuer issuing e-money tokens denominated in any given currency (numereraire).
+
+Here we summarise the anticipated behaviour and driving incentives of Issuers and Users and, what this means for our choice of functionality.
+
+## Context
+
+For simplicity of description, assume throughout we are discussing USD-denominated tokens.
+
+EMT = e-money token.
+
+Issuer behaviour:
+
+Issuers agree to support Fungible (at par) exchange and redemption of EMTs from other currently licensed issuers.
+
+Issuer incentives for the given behaviour:
+
+(a) Benefit: network effect leads to increased EMT use, with increased issuance and circulation.
+    (This is driven by the resulting User behaviour and incentives - below. )
+(b) Benefit: any given issuer would lose market share if they didn't agree to support fungility as stated.
+    (Exchange and redemption take their competitors' EMTs out of circulation, put their EMT into circulation.)
+(c) Cost: Issuers face Counterparty risk only while they hold competitors' EMTs - they can mitigate this by redeeming competitor tokens.
+
+User behaviour:
+
+(i) Users will treat EMTs from different Issuers are homogeneous.
+(ii) Users will accept wallets and related use cases that equally 'handle' EMTs of different Issuers as homogenous.
+     This will lead to enhanced 'ease of use' and, further adoption (see: Issuer incentives).
+
+## Decision
+
+We will update our functionality as follows:
+
+(1) Our chain will support only one EMT (per currency), issued by 1 or more Issuers.
+
+(2) Our chain's EMTs will all have the same given name? (Such as USD-E for USD.)
+
+(3) Added Issuer functionality.
+    We add a transaction type as follows:
+        For User U, licensed by Issuer I1, U can 'send' to I1 the EMT an Issuer I2, 'receiving' in returning the EMT of I.
+    The transaction will fail if:
+        I1 is up to its issuance limit or
+        The Issuer license of I2 has been revoked.
```

Comment:
```suggestion
Functionality will be updated as follows:

(1) Our chain will support only one EMT (per currency) that is issued by 1 or more Issuers.

(2) Our chain's EMTs will all have the same given name? For example, USD-E for USD.

(3) Added Issuer functionality to add a transaction type as follows:

    For User U, licensed by Issuer I1: 
    
    - U can 'send' to I1 the EMT an Issuer I2
    - U 'receiving' in returning the EMT of I

    The transaction will fail if:
    
    - I1 is up to its issuance limit or
    - The Issuer license of I2 has been revoked
```

Reply:
check line 57... is it U that receives? 

I suggest using the clear list item formatting as suggested here so that the functionality is more easily scanned and understood

Reply:
Adopted.